src/cli-translator/azext_cli_translator/custom.py

```diff
@@ -0,0 +1,42 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import os
+import json
+import requests
+from knack.util import CLIError
+
+ARM_TRANSLATOR_URL = 'https://portal2cli.azurewebsites.net/api/v1'
+
+
+def translate_arm(cmd, template_path, parameters_path, resource_group_name, target_subscription=None):
+    if not os.path.exists(template_path) or not os.path.exists(parameters_path):
+        raise CLIError('--template or --parameters file not found')
+
+    with open(template_path, 'r') as fp:
+        template_content = fp.read()
+    with open(parameters_path, 'r') as fp:
+        parameters_content = fp.read()
+    if not template_content or not parameters_content:
+        raise CLIError('--template or --parameters file is empty')
+
+    if target_subscription is None:
+        from azure.cli.core.commands.client_factory import get_subscription_id
+        target_subscription = get_subscription_id(cmd.cli_ctx)
+    try:
+        response = requests.post(
```

Comment:
Do we need authentication?

Reply:
We discussed it in the scrum meeting. No need currently.